service/src/main/kotlin/no/nav/su/se/bakover/service/utbetaling/UtbetalingServiceImpl.kt

```diff
@@ -113,6 +115,30 @@ internal class UtbetalingServiceImpl(
         }
     }
 
+    override fun genererUtbetalingsRequest(
```

Comment:
Kunne ikke denne ha vært privat? også kan det være `lagreUtbetaling` og `publiserUtbetaling` sitt ansvar å kalle denne?

Reply:
Problemet er at vi trenger resultatet (mer bestemt den genererte `id`-en) for å kunne lagre vedtak. Men kan sjekke opp om jeg får vridd det litt